### ISSUE TITLE
Wildcard CSS using box-sizing: border-box makes icons disappear on TreeView

### ISSUE THREAD

**Issue by Aaronius** [adobe/spectrum-css#530]
## Description

This issue might apply to more than just the TreeView component, but that's the one I saw an issue with. 

In our app, we would like to have either one of these CSS declaration blocks in our stylesheet:

```
* {
  box-sizing: border-box;
}
```

or

```
html {
  box-sizing: border-box;
}
*, *:before, *:after {
  box-sizing: inherit;
}
```

See https://css-tricks.com/inheriting-box-sizing-probably-slightly-better-best-practice/ for more details on the second option.

This is to make positioning elements consistently simpler. When this is added to a stylesheet and the TreeView is rendered, however, the disclosure icons disappear.

## Steps to reproduce

1. Go to https://react-spectrum.corp.adobe.com/storybook/?path=/story/treeview--default
2. Add `* { box-sizing: border-box; }` to the page using developer tools.
3. Observe that the disclosure icons disappear.

## Expected behavior
Preferably components would look the same as they would without the wildcard declaration block.

## Screenshots

![image](https://user-images.githubusercontent.com/210820/74276073-443e5400-4cd2-11ea-95ec-be45a96a4686.png)

## Environment
 - **Spectrum CSS version:** 2.0.5 I think?
 - **Browser(s) and OS(s):** Chrome 80.0.3987.87 Mac 10.14

## Additional context
Here's the relevant Slack conversation: https://adobe-launch.slack.com/archives/C7N5BE1EW/p1581375145196900

**Comment by Aaronius** [comment]
I'm not sure if you want to call this a bug or an enhancement.

**Comment by lazd** [comment]
Yup, it's a bug! Luckily, it's one we've already fixed: #467

**Comment by Aaronius** [comment]
🔥 🌶
